addons/website/static/src/snippets/s_numbers_showcase/000.scss

```
@@ -0,0 +1,4 @@
+.s_numbers_showcase:not([data-vcss]) .row > div {
```

Shouldn't we simply target `.o_grid_item` directly?  🤔

```suggestion
.s_numbers_showcase .o_grid_item {
```